integration-tests/soap/src/main/java/org/apache/camel/quarkus/component/soap/it/service/CustomerServiceService.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.camel.quarkus.component.soap.it.service;
+
+import java.net.URL;
+
+import javax.xml.namespace.QName;
+import javax.xml.ws.Service;
+import javax.xml.ws.WebEndpoint;
+import javax.xml.ws.WebServiceClient;
+import javax.xml.ws.WebServiceFeature;
+
+@WebServiceClient(name = "CustomerServiceService", wsdlLocation = "file:/home/james/Projects/git/master/camel/components/camel-soap/src/test/resources/org/apache/camel/dataformat/soap/CustomerService.wsdl", targetNamespace = "http://service.it.soap.component.quarkus.camel.apache.org/")
```

Comment:
I wonder if it would be possible to write something like wsdlLocation="classpath:org/apache/camel/dataformat/soap/CustomerService.wsdl" ?

Reply:
Ah good spot. I thought I caught all of the hard coded paths. I'll fix

Reply:
Latest commit removes the `wsdlLocation` annotation value as it's not actually needed in the context of this change.